docs/integrations/builtin/core-nodes/n8n-nodes-base.merge.md

```diff
@@ -7,96 +7,86 @@ contentType: integration
 
 # Merge
 
-Use the Merge node to combine data from two streams, once data of both streams is available.
+Use the Merge node to combine data from multiple streams, once data of all streams is available.
 
 /// note | Major changes in 0.194.0
 This node was overhauled in n8n 0.194.0. This document reflects the latest version of the node. If you're using an older version of n8n, you can find the previous version of this document [here](https://github.com/n8n-io/n8n-docs/blob/4ff688642cc9ee7ca7d00987847bf4e4515da59d/docs/integrations/builtin/core-nodes/n8n-nodes-base.merge.md){:target=_blank .external-link}.
 ///
 
-## Merge mode
+/// note | Minor changes in 1.49.0
+In n8n version 1.49.0, the option to add more than two inputs was introduced. Older versions only support up to two inputs. To combine multiple inputs in these versions, use the [Code node](/code/code-node/).
+The Mode > SQL Query feature was also added in n8n version 1.49.0 and isn't available in older versions.
+///
+
+## Node parameters
 
-You can specify how the Merge node should combine data from different branches. The following options are available:
+You can specify how the Merge node should combine data from different branches by choosing a **Mode**: 
 
 ### Append
 
-Keep data from both inputs. The output contains items from Input 1, followed by all items from Input 2.
+Keep data from all inputs. Choose a **Number of Inputs** to output items of each input, one after another. The node waits for the execution of all connected inputs. 
 
 ![Diagram](/_images/integrations/builtin/core-nodes/merge/append-diagram.png)
```

Comment:
```suggestion
<figure markdown="span">
![Sample Append mode inputs and output. Two separate data sources are on the left, one with items A, B, C and one with items D, E, F. The final data source combines both and lists A, B, C, D, E, F.](/_images/integrations/builtin/core-nodes/merge/append-diagram.png)
<figcaption>Sample Append mode inputs and output</figcaption>
</figure>
```

The Style Guide says to use alt text and captions but didn't have info on how to format them, I've updated there and provided a sample treatment here, @ayatnkw can you do similar to the other images in here, since we're updating this page anyway?